src/documentation/pages/components-wip/footer/examples/index.vue

```diff
@@ -0,0 +1,47 @@
+<template lang="pug">
+  div
+    h2 Examples
+    p.-text
+      | To use footers, apply the class <code>chi-footer</code> to <code>&lt;footer&gt;</code>
+      | and wrap all content in <code>chi-footer__content</code>.
+
+    div(v-if="['lumen', 'portal', 'brightspeed'].includes($store.state.themes.theme)")
+      chi-alert(color="info" icon="circle-info" title="Localization") 
+        | The Footer's language drop down is not functional by default as language support varies by project.
+        | Please ensure the drop down is populated with the correct languages and URL paths relevant to your project prior to release.
+
+    div(v-if="['lumen', 'portal'].includes($store.state.themes.theme)")
+      <ExternalLumenPortal />
```

Comment:
Can we do this with the rest?
```suggestion
      ExternalLumenPortal(v-if="['lumen', 'portal'].includes($store.state.themes.theme)")
```